src/Samples/NetCoreClient/Program.cs

```diff
@@ -6,16 +6,17 @@ namespace NetCoreClient
 {
     class Program
     {
-        private static readonly string s_hostname = "localhost";
-
-        static void Main()
+        /// <param name="hostname">use commanline argument --hostname localhost
+        /// or something similar to indicate the WCF Server hostname </param>
+        static void Main(string hostname)
         {
-            Settings settings = new Settings().SetDefaults(s_hostname, "EchoService");
+            Console.Title = "WCF .Net Core Client";
```

Comment:
If this is run on the command line, will this permanently change the console title or will it revert once the process exits?

Reply:
When the process exits, the console title reverts to the previous value